lib/index.js

```diff
@@ -12,13 +12,15 @@ function AuthPassportLocal () {
   this.context = {};
   this.pipes = pipes;
 
-  this.init = function (config, context, isDummy) {
+  this.init = function (customConfig, context, isDummy) {
     var strategy;
+    const defaultConfig = {
+      'secret': 'changeme',
```

Comment:
To me, the best way to ensure this, is to make the salt come down from Kuzzle config. We can safely assume that a cluster administrator is "able" to replicate consistent config across nodes. Or, do we need to be foolproof about that? Fool-administrated-clusters will have more sever problems IMHO.

That's why I keep on thinking that a simple, yet well visible, warning will do the bizness.